src/elements/widgets/designerView/designerCanvas.ts

```diff
@@ -864,7 +864,9 @@ export class DesignerCanvas extends BaseCustomWebComponentLazyAppend implements
         const containerStyle = getComputedStyle(newContainerElementDesignItem.element);
         newContainerService = this.serviceContainer.getLastServiceWhere('containerService', x => x.serviceForContainer(newContainerElementDesignItem, containerStyle));
         if (newContainerService) {
-          if (newContainerService.canEnterByDrop(newContainerElementDesignItem)) {
+          if (newContainerService.canEnterByDrop(newContainerElementDesignItem) &&
+            newContainerElementDesignItem.element.nodeName != 'svg' &&

```

Comment:
i would solve this in another way. 
also a path could be in a d wich could be in a svg. then your code would not work

you could check "instanceof SVGElement" this should be true for each type of element wich could be in a SVG